mergify_engine/actions/request_reviews.py

```diff
@@ -172,7 +172,9 @@ async def run(
         )
 
         if user_reviews_to_request or team_reviews_to_request:
-            requested_reviews_nb = len(await ctxt.pull_request.review_requested)
+            requested_reviews_nb = len(
+                typing.cast(typing.List[str], await ctxt.pull_request.review_requested)
```

Comment:
I wonder if we can do better for  `PullRequest.__getattr__, I don't really like the casting each time we use an attribute.

Reply:
Yes, I thought about it. We should add something like a `@property` for each field that is exposed on the pull request object.